.github/workflows/maven.yml

```diff
@@ -271,6 +273,88 @@ jobs:
           fi
           ./scripts/check_histogram_results.sh results/histograms.json $ERRORS_THRESHOLD
 
+  ## ----------------------------------------------------------------------------------
+  ## ORACLE
+  ## ----------------------------------------------------------------------------------
+  oracle:
+    needs: package-and-upload
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        benchmark: [ 'auctionmark', 'epinions', 'hyadapt', 'otmetrics', 'resourcestresser', 'seats', 'sibench', 'smallbank', 'tatp', 'tpcc', 'twitter', 'voter', 'wikipedia', 'ycsb', 'templated' ]
+    services:
+      oracle:
+        image: gvenzl/oracle-xe:21.3.0-slim-faststart
```

Comment:
Is this the officially supported Oracle docker image?

Reply:
This image is publicly and freely available via https://github.com/gvenzl/oci-oracle-free. It’s also used in other open-source projects like this Micronaut example application (https://blogs.oracle.com/java/post/json-relational-duality-views-with-micronaut-framework) and by the  testcontainers framework (https://java.testcontainers.org/modules/databases/oraclexe/).  The Oracle official container registry can be found here: https://container-registry.oracle.com/.

Reply:
I'm not huge Oracle expert.
Is there a particular reason not to use the `container-registry.oracle.com/database/free:latest` image?
My intuition is that that would be more representative of a real Oracle DB for use in testing at least and is the officially supported upstream version.
What advantages or maintenance does the one provided by gvenzl offer?

Reply:
The gvenzl image is slimmer (`gvenzl/oracle-xe:21.3.0-slim-faststart` at 4.63GB vs `container-registry.oracle.com/database/free:23.2.0.0` at 10.60GB through `docker inspect`) and starts slightly faster than official Oracle DB.

It also offers better maintenance. gvenzl image offers extra options `APP_USER` and `APP_USER_PASSWORD` to create a user on start up (similar to MySQL `APP_USER` and `APP_PASSWORD`) which is not supported by official Oracle DB image. On official Oracle image, in order to create the user I'd need to mount `reset.sql` to startup instead of setup due to an [open issue](https://github.com/oracle/docker-images/issues/2644).